core/src/solvers/joint_interpolation.cpp

```diff
@@ -78,10 +78,10 @@ PlannerInterface::Result JointInterpolationPlanner::plan(const planning_scene::P
 	// add first point
 	result->addSuffixWayPoint(from->getCurrentState(), 0.0);
 	if (from->isStateColliding(from_state, jmg->getName()))
-		return { false, "Start state is in collision!" };
+		return MoveItErrorCode(MoveItErrorCodes::START_STATE_IN_COLLISION, "Start state is in collision!");
```

Comment:
These returned in `ros-old` just FAILURE, I changed them so that the error code actually provides value

Reply:
Note, that the error code is never actually considered in code - we just check for success. 
It might be useful for inspection during debugging though.

Reply:
Yes, we did that in some custom applications. Given that MoveIt generates this error code, it would be great not to loose this information in the MTC solution.

Reply:
How did you access that code in custom applications? It is never exposed from any stage!?

Reply:
You're right, I think was wrong about that, sorry